config/autoload/zend-expressive.global.php

```diff
@@ -3,9 +3,6 @@
 return [
     'debug' => false,
 
-    // Cache the configuration. Recommended for production.
-    'config_cache_enabled' => false,
```

Comment:
> One of the key reasons for having development mode was to solve the problem of ensuring it's optimized by default, but can be development ready otherwise. 

Makes sense. So where to set that boolean to true? In config\autoload\zend-expressive.global.php?